js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -89,7 +89,9 @@ const ValidatingController = modules.Controller.inherit((function() {
             var brokenRulesMessages = [];
 
             each(brokenRules, function(_, brokenRule) {
-                if(!brokenRule.validator.$element().parent().length) {
+                let isVisibleColumn = brokenRule.column && brokenRule.column.visible && brokenRule.column.groupIndex === undefined;
```

Comment:
Change 
`brokenRule.column.groupIndex === undefined`
to
`!brokenRule.column.type`

because the showWhenGrouped option can be enabled in column.